airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -36,7 +36,7 @@
 from airflow.models import BaseOperator, BaseOperatorLink
 from airflow.models.xcom import XCom
 from airflow.operators.sql import SQLCheckOperator, SQLIntervalCheckOperator, SQLValueCheckOperator
-from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook, BigQueryJob
+from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook, BigQueryJob, _split_tablename
```

Comment:
importing private function is usually bad practice isn't it?

Reply:
Thanks for the comment. Indeed, private functions are not meant to be imported. But there is no in-class function that could split the table name, which means I either import the private function or rewrite it in the current class. I suppose the former is more acceptable?

Also, I do see that the code imported some other private functions as shown below.
https://github.com/apache/airflow/blob/7fed7f31c3a895c0df08228541f955efb16fbf79/airflow/providers/google/cloud/operators/bigquery.py#L39-L41

Reply:
It might be useful to ask the upstream project why this function is private (in the first place).

Reply:
It's in our project. The function is defined in `BigQueryHook`:
https://github.com/apache/airflow/blob/b1ad017cee66f5e042144cc7baa2d44b23b47c4f/airflow/providers/google/cloud/hooks/bigquery.py#L2770

Reply:
I think it's ok. We can use it in the project as we see fit but we basically tell the users don't use the function in custom operators. As for why it's marked private to begin with - I can't tell... I can assume that it's because this is a helper function that takes part in "internal" functionality of operators so the author of the function didn't expect users to use it directly in thier operators.

Reply:
Seems like https://github.com/apache/airflow/pull/24416/files is changing this function.
so lets wait for this PR to be merged first